client/src/pages/index.js

```diff
@@ -1,10 +1,20 @@
 import React from "react";
 import Layout from "../components/Layout";
+import Logo from '../components/Common/Logo';
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  padding: 8px 0;
+  width: 2cm;
+`;
 
 const MainPage = () => {
   return (
     <Layout>
       <div>This will be the home page</div>
+      <Wrapper>
```

Comment:
For now, this basic "Wrapper" is fine, but eventually we will have more components. At that point "Wrapper" is very generic, and we will want to update to "LogoWrapper" or something like that.

Reply:
This was just for demo purposes, but I'll fix that up